test/test_sync.cpp

```diff
@@ -5733,6 +5733,56 @@ TEST(Sync_BadChangeset)
 }
 
 
+TEST(Sync_GoodChangeset_AccentCharacterInFieldName)
+{
+    TEST_DIR(dir);
+    TEST_CLIENT_DB(db);
+
+    bool did_fail = false;
+    {
+        ClientServerFixture::Config config;
+        config.disable_upload_compaction = true;
+        ClientServerFixture fixture(dir, test_context, std::move(config));
+        fixture.start();
+
+        {
```

Comment:
Do we need to do this step where we wait for download complete? I think this should be a no-op.